feconf.py

```diff
@@ -169,6 +169,16 @@ def get_empty_ratings():
 # Empty scaled average rating as a float.
 EMPTY_SCALED_AVERAGE_RATING = 0.0
 
+# To use GAE email service.
+GAE_EMAIL_SERVICE = 'gae_email_service'
```

Comment:
For constants of a particular type, start with the type first -- so, EMAIL_SERVICE_PROVIDER_GAE, EMAIL_SERVICE_PROVIDER_MAILGUN.